report.php

```diff
@@ -56,6 +67,22 @@
     echo $OUTPUT->heading($title);
 }
 
+
+$action = new \confirm_action(get_string('areyousuretodeleteallrecords', 'tool_curlmanager'));
+$urlresetallcspstatistics = new moodle_url($PAGE->url, array(
+    'resetallcurlstatistics' => 1,
+    'sesskey' => sesskey(),
+));
+echo $OUTPUT->single_button($urlresetallcspstatistics,
+                get_string('resetallcurlstatistics', 'tool_curlmanager'),
+                'post',
+                [
+                    'actions' => [$action],
+                    'style' => 'float: right;'
```

Comment:
I don't think this works (at least in moodle), and either way please remove the float

Reply:
No this didn't work. I have removed float right.